app.js

```diff
@@ -8,56 +8,90 @@ if (responseOne){
+    console.log('Awesome! You are correct!');
+  } else {
+    console.log('Too bad, you were wrong. Anna has a dog.');
+  }
 }
 
-var quizResponseTwo = prompt('Please answer with Y or N. Is Anna from Seattle?');
+function questionTwo() {
+  var quizResponseTwo = prompt('Please answer with Y or N. Is Anna from Seattle?');
 
-if (quizResponseTwo.toUpperCase() === 'N' || quizResponseTwo.toUpperCase() === 'NO'){
-  console.log('Awesome! You are correct!');
-} else {
-  console.log('Too bad, you were wrong. Anna is from Louisiana.');
+  if (quizResponseTwo.toUpperCase() === 'N' || quizResponseTwo.toUpperCase() === 'NO'){
+    console.log('Awesome! You are correct!');
+  } else {
+    console.log('Too bad, you were wrong. Anna is from Louisiana.');
+  }
 }
 
-var quizResponseThree = prompt('Please answer with Y or N. Does Anna like owls?');
+function questionThree() {
+  var quizResponseThree = prompt('Please answer with Y or N. Does Anna like owls?');
 
-if(quizResponseThree.toUpperCase() === 'Y' || quizResponseThree.toUpperCase() === 'YES'){
-  console.log('Awesome! You are correct!');
-} else {
-  console.log('Too bad. You were wrong.');
+  if(quizResponseThree.toUpperCase() === 'Y' || quizResponseThree.toUpperCase() === 'YES'){
+    console.log('Awesome! You are correct!');
+  } else {
+    console.log('Too bad. You were wrong.');
+  }
 }
 
-var quizResponseFour = prompt('Please answer with Y or N. Does Anna like to read?');
+function questionFour() {
+  var quizResponseFour = prompt('Please answer with Y or N. Does Anna like to read?');
 
-if(quizResponseFour.toUpperCase() === 'Y' || quizResponseFour.toUpperCase() === 'YES'){
-  console.log('Awesome! You are correct!');
-} else {
-  console.log('Too bad. You were wrong.');
+  if(quizResponseFour.toUpperCase() === 'Y' || quizResponseFour.toUpperCase() === 'YES'){
+    console.log('Awesome! You are correct!');
+  } else {
+    console.log('Too bad. You were wrong.');
+  }
 }
 
-var quizResponseFive = prompt('Please answer with Y or N. Does Anna like the cold weather here in Seattle?');
+function questionFive() {
+  var quizResponseFive = prompt('Please answer with Y or N. Does Anna like the cold weather here in Seattle?');
 
-if(quizResponseFive.toUpperCase() === 'N' || quizResponseFive.toUpperCase() === 'NO'){
-  console.log('Awesome! You are correct!');
-} else {
-  console.log('Too bad. You were wrong. Anna dislikes the cold weather very much!');
+  if(quizResponseFive.toUpperCase() === 'N' || quizResponseFive.toUpperCase() === 'NO'){
+    console.log('Awesome! You are correct!');
+  } else {
+    console.log('Too bad. You were wrong. Anna dislikes the cold weather very much!');
+  }
 }
 
-var quizResponseSix = prompt('How many dogs do I have? You have four guesses to get it right!');
-
-if(quizResponseSix = 2) {
-  congratulate('Yay! You got it right!');
-} else {
-  encourage('Bummer! Try Again.');
+function questionSix() {
+  for (var i = 0; i < 4 ; i++) {
```

Comment:
What is the number four? Unexplained numbers like this are typically called "magic numbers," because no one other than the developer who wrote the code initially know what they are. It's better practice to store the number in a variable so that you can label what the number will be used for, like this: 
```javascript
var numberOfTries = 4;
for (var i = 0; i < numberOfTries; i++) {
```